lib/service/config.ts

```diff
@@ -39,28 +49,72 @@ export default class Config {
 
   set(config) {
     // FIXME: Deep copy to avoid deleting itself.
-    config = JSON.parse(JSON.stringify(config))
+    const newConfig = { ...config }
```

Comment:
Maybe this change will cause this problem. Would you like to undo this change and work on this ticket? We can make it so simple like the following if we worked on it.
https://github.com/crowi/crowi/pull/475

```js
set(config) {
  this.config = config
}
```